search-results.hbs

```diff
@@ -1,154 +1,215 @@
-{{!< default}}
-{{!-- The tag above means - insert everything in this file into the {body} of the default.hbs template --}}
+<!DOCTYPE html>
+<html lang="{{@site.lang}}">
+
+<head>
+
+    {{!-- Document Settings --}}
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    {{!-- Base Meta --}}
+    <title>Search Results</title>
+    <meta name="HandheldFriendly" content="True" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    {{!-- Styles'n'Scripts --}}
+    <link rel="stylesheet" type="text/css" href="{{asset "built/screen.css"}}" />
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,400i,700|Roboto+Mono:400,700" rel="stylesheet">
+
+    {{!-- Styles'n'Scripts --}}
+    <link rel="stylesheet" type="text/css" href="{{asset "built/screen.css"}}" />
+    <link rel="stylesheet" type="text/css" href="{{asset "css/algolia-search.css"}}" />
+
+    {{!-- Algolia search assets --}}
+    <script src="https://cdn.freecodecamp.org/news-assets/algolia/algoliasearch-3-33-0/algoliasearch.min.js"></script>
+    <script src="https://cdn.freecodecamp.org/news-assets/algolia/autocomplete-0-36-0/autocomplete.min.js"></script>
+
+    {{!-- Add/update missing meta properties for Twitter and Facebook cards --}}
+    <meta name="twitter:title" content="Search Results" />
+    <meta name="twitter:description" content="Learn to code at home. Build projects. Earn certifications. 
+    Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including 
+    Google, Apple, Amazon, and Microsoft." />
+    <meta name="twitter:url" content="https://www.freecodecamp.org/news/search/" />
+    <meta name="twitter:image" content="https://www.freecodecamp.org/news/content/images/2019/11/fcc_ghost_publication_cover.png" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@tryghost" />
+
+    <meta property="og:title" content="Search Results" />
+    <meta property="og:description" content="Learn to code at home. Build projects. Earn certifications. 
+    Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including 
+    Google, Apple, Amazon, and Microsoft." />
+    <meta property="og:url" content="https://www.freecodecamp.org/news/search/" />
+    <meta property="og:image" content="https://www.freecodecamp.org/news/content/images/2019/11/fcc_ghost_publication_cover.png" />
```

Comment:
Should we have these in a CDN?
It seems that this link would break if the original file is removed or modified.

Reply:
If I remember correctly, I got this from the publication cover section of the Ghost settings page. I could make use a CDN link if that's better.

Reply:
yes, I believe that is how we are handling them on /learn

Reply:
Sure thing, I can do that and update this PR.